src/components/dropdown.tsx

```
@@ -0,0 +1,42 @@
+}
+
+const Dropdown = ({ size = "md", items }: DropdownProps) => {
+  return (
+    <div
+      className={`absolute ${
+        size === "sm" ? "w-[81px] h-auto" : "w-[106px] h-auto"
+      } shadow-lg bg-white rounded-[12px] overflow-hidden`}
+    >
+      {items.map((item, index) => (
+        <button
+          key={index}
```

ESLint

[error] Do not use Array index in keys

(react/no-array-index-key)